config/crd/bases/multicluster.odf.openshift.io_mirrorpeers.yaml

```diff
@@ -66,6 +66,13 @@ spec:
               manageS3:
                 default: false
                 type: boolean
+              overlappingCIDR:
+                default: false
+                description: OverlappingCIDR should be set to true if the peer clusters
+                  have overlapping Pod or Service CIDR. This will enable storage clusters
+                  to use submariner globalnet. Enabling this will cause storage service
+                  disruption while network is being reconfigured.
```

Comment:
The changes look good. But again checking if we can have the multiClusterSerivice enabled when the ODF cluster is first deployed because then we don't have to wait while the network is being reconfigured.

Reply:
It's possible, but requires some changes in ODF UI/UX.